### PR TITLE
Add, Update and Rearrange service groups community docs and folders

### PR DESCRIPTION
**Pull request by schwesig**
Update sigs.yaml
Update governance.md
Create README.md
Create sig/README.md
Create wg/README.md
Create sub-projects.md
Create working-groups.md
Update sig-list.md
Create charter.md
Create sig-charter-template.md
Create sig-governance-requirements.md
Create sig-governance.md

### CODE REVIEW

**Comment by schwesig** [comment]
@goern

File: README.md [hunk 1]
```diff
@@ -6,35 +6,35 @@ This is the starting point for joining and contributing to the Open Services Gro
...
 
-The way how we organise our community and work is heavily inspired by the Kubernetes Community!
+The way how we organise our community and work is heavily inspired by the [Kubernetes Community]!
 
 ## Governance
 
 Open Services Group has the following types of groups that are officially supported:
 
-* **Steering Committee** are named people that are chartered to take on sensitive topics.
+* **[Steering Committee](/sig-list.md)** are named people that are chartered to take on sensitive topics.
```
Comment (durandom) [inline README.md]: should this be a different file than the sig-list? A `steering-list.md`

**Review by durandom — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (schwesig) [reply README.md]: yes, thanks, done: committee-steering/README.md

**Pull request merged**